src/main.lua

```diff
@@ -1206,8 +1215,8 @@ end)
 addEventingHandler(ActionMap.CancelWithdrawal, utils.hasMatchingTag("Action", ActionMap.CancelWithdrawal), function(msg)
 	local gatewayAddress = msg.Tags.Target or msg.Tags.Address or msg.From
 	local vaultId = msg.Tags["Vault-Id"]
-	assert(utils.isValidAOAddress(gatewayAddress), "Invalid gateway address")
-	assert(utils.isValidAOAddress(vaultId), "Invalid vault id")
+	assert(utils.isValidAddress(gatewayAddress), "Invalid gateway address")
```

Comment:
these are laxed bc to join the network `msg.From` will be used for the gateway address. if AO introduces support for new addresses, we don't want to worry about blocking delegations on those new addresses. the check for the `gateway` itself is sufficient.

Reply:
Is it actually relaxed if you're not passing "allow unsafe" here?

Reply:
🤔